index.js

```diff
@@ -57,7 +57,9 @@ function writeFile(filepath, data, options, cb) {
       cb(err);
       return;
     }
-    fs.writeFile(filepath, data, options, cb);
+
+    var withEndNewLine = data.slice(-1) === "\n" ? data : data + "\n";
```

Comment:
I think we will need to expose an option for this since it changes behavior. I remember doing this on another lib and it broke some downstream implementations, so we exposed an option for it. I'm trying to remember which lib that was...

Like this:

```js

if (options && options.ensureNewline === true && data.slice(-1) !== '\n') {
  data += '\n';
}
```

We'll also need to add this to all methods.

Reply:
Sure, I'll take a look on it tonight. :)